<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracked camera client software</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <widget class="QWidget" name="cw">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="CaptureWgt" name="view" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="cmdDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Command window</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QtLua::Console" name="console" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="btnsDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QPushButton" name="lightBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Light</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="motoEnBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Moto en</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="fwdBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>12</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bwdBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>40</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>40</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>40</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="queryStatus"/>
   <addaction name="dontSleep"/>
   <addaction name="queryOsc"/>
   <addaction name="clearLog"/>
   <addaction name="showFullLog"/>
   <addaction name="separator"/>
   <addaction name="shutdown"/>
   <addaction name="separator"/>
   <addaction name="setupPipe"/>
  </widget>
  <widget class="QDockWidget" name="statusDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Status</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="status"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="showFullLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show full log</string>
   </property>
   <property name="toolTip">
    <string>Show full log</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="shutdown">
   <property name="text">
    <string>Shutdown</string>
   </property>
   <property name="toolTip">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="showLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show log</string>
   </property>
   <property name="toolTip">
    <string>Show log</string>
   </property>
  </action>
  <action name="queryStatus">
   <property name="text">
    <string>Status</string>
   </property>
   <property name="toolTip">
    <string>Query status</string>
   </property>
  </action>
  <action name="clearLog">
   <property name="text">
    <string>Clear log</string>
   </property>
   <property name="toolTip">
    <string>Clear log</string>
   </property>
  </action>
  <action name="queryOsc">
   <property name="text">
    <string>Osc</string>
   </property>
   <property name="toolTip">
    <string>Query oscilloscopic measures</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="dontSleep">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Don`t sleep</string>
   </property>
   <property name="toolTip">
    <string>Send Don't sleep periodically to prevent robot hybernation.</string>
   </property>
  </action>
  <action name="setupPipe">
   <property name="text">
    <string>Setup pipe</string>
   </property>
   <property name="toolTip">
    <string>Setup pile to remote host</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtLua::Console</class>
   <extends>QWidget</extends>
   <header>QtLua/Console</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptureWgt</class>
   <extends>QWidget</extends>
   <header>capture_wgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
